packages/good-design/src/core/layout/Trans.tsx

```diff
@@ -1,22 +1,31 @@
 import React from "react";
 import { Trans as LinguiTrans } from "@lingui/react";
 
-import { IButtonProps, ITextProps, Text } from "native-base";
+import { Heading, IButtonProps, ITextProps, IHeadingProps, Text } from "native-base";
 import Title from "./Title";
 import { GoodButton } from "../buttons";
 
 /**
  * Use this component when wanting to apply Text variants (see theme.tsx)
- * @param t text that needs to supported translation
+ * @param t text that needs to be translated
  * @returns Text
  */
 export const TransText = ({ t, ...props }: { t: string } & ITextProps) => (
   <LinguiTrans id={t} render={({ translation }: { translation: any }) => <Text {...props}>{translation}</Text>} />
 );
 
+/**
+ * Use this component when wanting to apply Text variants (see theme.tsx)
+ * @param t text that needs to be translated
+ * @returns Heading
+ */
+export const TransHeading = ({ t, ...props }: { t: string } & IHeadingProps) => (
+  <LinguiTrans id={t} render={({ translation }: { translation: any }) => <Heading {...props}>{translation}</Heading>} />
```

Comment:
why do we need to use the render=
why cant we use simple `<heading><trans t='label'/></heading>`

Reply:
Actually no particular reason that I can think of.
Works with your given example as well, applied fix here: https://github.com/GoodDollar/GoodWeb3-Mono/pull/171/commits/ffb8b9a58fe681229c9b7255682c9323840aa1fb